src/components/Header/Header.jsx

```diff
@@ -1,84 +1,84 @@
-import React from 'react';
-import { Box } from '@mui/system';
-import Button from '@mui/material/Button';
-import Avatar from '@mui/material/Avatar';
-import Typography from '@mui/material/Typography';
-import { Stack } from '@mui/material';
-import InputBase from '@mui/material/InputBase';
-import client from '../../utils/client';
-import { useNavigate } from 'react-router-dom';
+import React from "react"
+import { Box } from "@mui/system"
+import Button from "@mui/material/Button"
+import Avatar from "@mui/material/Avatar"
+import Typography from "@mui/material/Typography"
+import { Stack } from "@mui/material"
+import InputBase from "@mui/material/InputBase"
+import { useNavigate } from "react-router-dom"
+import { Link } from "react-router-dom"
 
 const Header = ({ role, userId }) => {
-  let navigate = useNavigate();
+  let navigate = useNavigate()
 
   const signOut = (event) => {
-    event.preventDefault();
-    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, '');
-    navigate('../login', { replace: true });
-  };
-
-  const addCohortHandle = () => {
-    client.post('/cohort', {}).catch((err) => console.log(err.response));
-  };
+    event.preventDefault()
+    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, "")
+    navigate("../login", { replace: true })
+  }
 
   const handleMyProfileLink = () => {
-    const userId = localStorage.getItem('userId');
-    navigate(`../user/${userId}`);
-  };
+    const userId = localStorage.getItem("userId")
```

Comment:
Can we change the strings in this file to use single quotes and have semi-colons at the end of each line, please?